opennms-base-assembly/src/main/filtered/etc/create.sql

```diff
@@ -1122,6 +1123,12 @@ CREATE TABLE alarm_attributes (
 CREATE INDEX alarm_attributes_idx ON alarm_attributes(alarmID);
 CREATE UNIQUE INDEX alarm_attributes_aan_idx ON alarm_attributes(alarmID, attributeName);
 
+CREATE TABLE alarm_situations (
+    situation_id    INTEGER,
+    alarms_alarmid  INTEGER
+);
```

Comment:
The changelog.xml and the create.sql show differences, e.g. some constraints are missing. 
This may result in tests pass, but in live scenario it may not work.
Can we add the missing restrictions here as well?

Reply:
Thanks. I've updated the test SQL to be in line with the liquibase schema.

Reply:
👍 Thank you